doc/api/errors.md

```diff
@@ -1957,6 +1957,12 @@ An attempt was made to load a module with an unknown or unsupported format.
 An invalid or unknown process signal was passed to an API expecting a valid
 signal (such as [`subprocess.kill()`][]).
 
+<a id="ERR_UNSUPPORTED_ESM_URL_SCHEME"></a>
+### `ERR_UNSUPPORTED_ESM_URL_SCHEME`
+
+An ES module has been attempted to be imported from a URL that is not
```

Comment:
I'd suggest this wording instead:

```suggestion
`import` with URL schemes other than `file` and `data` is unsupported.
```

...and delete the next line.

Reply:
Sure. :+1: